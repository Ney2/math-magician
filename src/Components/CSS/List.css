* {
  margin: 0;
  padding: 0;
}

.nav-header {
  display: flex;
  justify-content: space-around;
  background-color: #fff;
  align-items: baseline;
  height: 80px;
  font-size: large;
}

.nav-header h1 {
  align-self: baseline;
  color: green;
  margin-top: 1%;
}

.nav-link {
  text-decoration: none;
  background-color: #fff;
  margin-left: 30%;
  color: green;
  font-size: larger;
}

.nav-link:focus {
  color: red;
  text-decoration: underline;
}

@media (max-width: 768px) {
  body {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
  }

  .nav-header {
    width: fit-content;
  }
}
