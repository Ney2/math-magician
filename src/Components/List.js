import React from 'react';
import { NavLink } from 'react-router-dom';
import './CSS/List.css';

function List() {
  return (
    <header className="nav-header">
      <h1>Math Magicians</h1>
      <nav className="nav-bar">
        <div className="nav-conatiner">
          <NavLink className="nav-link" to="/">Home</NavLink>
          <NavLink className="nav-link" to="/calculator">Calculator</NavLink>
          <NavLink className="nav-link" to="/quotes">Quote</NavLink>
        </div>
      </nav>
    </header>
  );
}

export default List;
