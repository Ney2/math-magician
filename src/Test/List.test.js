/* eslint-disable no-undef */
import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import Renderer from 'react-test-renderer';
import List from '../Components/List';

function MockHeader() {
  return (
    <BrowserRouter>
      <List />
    </BrowserRouter>
  );
}

it('Renders correctly', () => {
  const tree = Renderer
    .create(<MockHeader />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
