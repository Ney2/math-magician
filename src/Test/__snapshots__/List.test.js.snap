// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<header
  className="nav-header"
>
  <h1>
    Math Magicians
  </h1>
  <nav
    className="nav-bar"
  >
    <div
      className="nav-conatiner"
    >
      <a
        aria-current="page"
        className="nav-link active"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="nav-link"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        className="nav-link"
        href="/quotes"
        onClick={[Function]}
      >
        Quote
      </a>
    </div>
  </nav>
</header>
`;
